test/node_roles_functional/background_tasks.config.ts

```diff
@@ -40,7 +40,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         '--usageCollection.usageCounters.bufferDuration=0',
 
         `--plugin-path=${path.resolve(__dirname, 'plugins', 'core_plugin_initializer_context')}`,
-        '--node.roles=["background_tasks"]',
+        '--node.roles=["ui","background_tasks"]',
```

Comment:
I guess this change is needed because, otherwise, we'll never run migrations. Correct?
Should we use the option `skipMigrations: true` instead?

Reply:
migrations.skip also never runs the migrations, so even though it won't block startup nothing creates the kibana indices and then task manager throws unhandled rejections killing the process.

Reply:
Makes sense, it just means that at this point the `background_tasks.config.ts` and `all.config.ts` tests are basically testing the same thing, so we probably don't need to keep running both of them.

Reply:
@lukeelmers I was just reviewing everything under `test/node_roles_functional` and I noticed
```
      await supertest.get('/core_plugin_initializer_context/node/roles').expect(200, {
        backgroundTasks: true,
        ui: true,
      });
```

I was thinking that ui flag should be false or this test should be dropped as kinda mentioned above.

 I can put a pr for this, what say you mate? :)

Reply:
Ahh, maybe I'm wrong: https://github.com/elastic/kibana/blob/main/test/node_roles_functional/plugins/core_plugin_initializer_context/server/plugin.ts#L23